tests/integration/study_data_blueprint/test_binding_constraints.py

```diff
@@ -630,82 +537,48 @@ def test_lifecycle__nominal(self, client: TestClient, user_access_token: str, st
         res = client.put(
             f"/v1/studies/{study_id}/bindingconstraints/binding_constraint_2",
             json={"less_term_matrix": [[]]},
-            headers=user_headers,
         )
         assert res.status_code == 422, res.json()
         assert res.json()["exception"] == "InvalidFieldForVersionError"
         assert res.json()["description"] == "You cannot fill a 'matrix_term' as these values refer to v8.7+ studies"
 
     @pytest.mark.parametrize("study_type", ["raw", "variant"])
-    def test_for_version_870(self, client: TestClient, admin_access_token: str, study_type: str) -> None:
-        admin_headers = {"Authorization": f"Bearer {admin_access_token}"}
+    def test_for_version_870(self, client: TestClient, user_access_token: str, study_type: str) -> None:
+        client.headers = {"Authorization": f"Bearer {user_access_token}"}  # type: ignore
```

Comment:
Why reduce user permissions (going from admin to user?)

Reply:
This is a bug: we must use the "user" token instead of the "admin" token